main.cpp

```diff
@@ -37,27 +37,166 @@ Purpose:  This project continues developing Project3 and will take you through h
  7) After you finish, click the [run] button.  Clear up any errors or warnings as best you can.
  
  */
-
+#include<iostream>
 /*
  copied UDT 1:
  */
+struct DjMixer
+{
+    float channelselected;
+    int volumeOne;
+    int volumeTwo;
+
+    void outputMusic();
+    void increaseVTwo();
+    void increaseVOne();
+    void switchChannelRight();
+    DjMixer();
+    ~DjMixer()
+    {
+        std::cout << "DjMixer Destructor." << std::endl;
+    }
+};
+
+DjMixer::DjMixer():channelselected(1.5f){}
 
+void DjMixer::switchChannelRight()
+{
+    while(channelselected <= 2.0f)
+    {
+        ++ channelselected;
+        if(channelselected == 2.0f)
+        {
+            std::cout << "Right Channel" << std::endl;
+        }
+    }
+}
 /*
  copied UDT 2:
  */
+struct Synth
+{
+    int oscillators;
+    bool reverb;
+    int reverbSync;
+    int syncBPM;
+    bool delay;
 
+    void reverbTime();
+    void selectDelay();
+    void selectBPM();
```

Comment:
Provide the implementation for these member function outside of the class
- [ ] revise